### PR TITLE
AIP - wstETH, MAI, stMATIC supply caps update Polygon/Arbitrum v3 

### PR DESCRIPTION
**Pull request by yonikesel**
discussions:
https://governance.aave.com/t/arfc-supply-and-borrow-caps-update-wsteth-v3-arbitrum/12309
https://governance.aave.com/t/arfc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-2023-03-16/12310

### CODE REVIEW

File: content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Supply/Borrow Cap Updates V3 Polygon and Arbitrum
+shortDescription: Update Supply and Borrow Caps on Aave V3 Polygon and Arbitrum
+author: Chaos Labs (@ori-chaoslabs, @yonikesel)
+discussions: https://governance.aave.com/t/arfc-supply-and-borrow-caps-update-wsteth-v3-arbitrum/12309, https://governance.aave.com/t/arfc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-2023-03-16/12310
```
Comment (sakulstra) [inline content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md:5]:
```suggestion
discussions: https://governance.aave.com/t/arfc-supply-and-borrow-caps-update-wsteth-v3-arbitrum/12309
```
I think currently non of the existing uis support multiple discussions, better to just link one and add both on references

File: content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+| Asset   | Current Supply Cap | Recommended Supply Cap | Current Borrow Cap | Recommended Borrow Cap |
+| ------- | ------------------ | ---------------------- | ------------------ | ---------------------- |
+| MAI     | 1,100,000          | 2,200,000              | 600,000            | 1,200,000              |
+| stMATIC | 15,000,000         | 21,000,000             | N/A                | N/A                    |
+
+# References
+
+Forum Posts: [Arbitrum](https://governance.aave.com/t/arfc-supply-and-borrow-caps-update-wsteth-v3-arbitrum/12309), [Polygon](https://governance.aave.com/t/arfc-chaos-labs-supply-and-borrow-cap-updates-aave-v3-polygon-2023-03-16/12310)
+
+Tests: [Arbitrum-stETH](https://github.com/bgd-labs/aave-proposals/blob/master/src/test/arbitrum/AaveV3ArbwstETHSupplyCapsPayloadTest-Mar16.t.sol), [Polygon-MAI](https://github.com/bgd-labs/aave-proposals/blob/master/src/test/polygon/AaveV3PolMAICapsPayloadTest-Mar16.t.sol), [Polygon-stMATIC](https://github.com/bgd-labs/aave-proposals/blob/master/src/test/polygon/AaveV3PolSTMATICCapPayloadTest-Mar17.t.sol)
```
Comment (sakulstra) [inline content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md:43]: I think these links are wrong, the test is called [AaveV3PolCapsPayloadTest-Mar17.t.sol](https://github.com/bgd-labs/aave-proposals/pull/98/files#diff-152eaed569ddcf22b26a6253f7d39ce6e8c0707e28bb7bfa81138cc4278d125d)  and includes both mai and stETH

File: content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Proposal payload implementation: [Arbitrum-stETH](https://github.com/bgd-labs/aave-proposals/blob/master/src/contracts/arbitrum/AaveV3ArbwstETHSupplyCapsPayload-Mar16.sol), [Polygon-MAI](https://github.com/bgd-labs/aave-proposals/blob/master/src/contracts/polygon/AaveV3PolMAICapsPayload-Mar16.sol), [Polygon-stMATIC](https://github.com/bgd-labs/aave-proposals/blob/master/src/contracts/polygon/AaveV3PolSTMATICCapPayload-Mar17.sol)
```
Comment (sakulstra) [inline content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md:45]: implementation links are wrong as well

File: content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md (hunk 2 above)
Comment (yonikesel) [reply content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md]: fixed.

File: content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md (hunk 3 above)
Comment (yonikesel) [reply content/aips/AIP-SUPPLY-CAPS-POLYGON-ARBITRUM-V3.md]: fixed.

**Pull request merged**